### ISSUE TITLE
Feature request: send raw tx

### ISSUE THREAD

**Issue by amanusk** [xJonathanLEI/starkli#58]
As a complementary feature to #57, it would be great to have the option to broadcast a raw tx by only providing a network/RPC provider. 
Useful if you want to modify data created by `--raw-tx` manually, and then only use starkli as a convenient interface to broadcast the raw JSON data (e.g. by supplying a file or a string)

**Comment by xJonathanLEI** [comment]
This feature makes a lot of sense once #57 is implemented. However, regarding the ``modify data created by `--raw-tx` manually`` part I'm not so sure.. If you manually modify data, the signature would change right? Then you cannot really broadcast it anymore.

**Comment by amanusk** [comment]
Good point, in our usecase we indeed modified both data and signature. I still think this is useful to have this option as an easy way to send (or resend) raw txs
